packages/snaps-controllers/src/services/webview/SnapsContext.ts

```diff
@@ -0,0 +1,4 @@
+export const snapsContext = {
```

Comment:
What is the goal of this file? Feels like this perhaps should be a constructor argument?